src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/INuGetUIContext.cs

```diff
@@ -37,5 +39,7 @@ public interface INuGetUIContext : IDisposable
         Task<bool> IsNuGetProjectUpgradeableAsync(IProjectContextInfo project, CancellationToken cancellationToken);
 
         Task<IModalProgressDialogSession> StartModalProgressDialogAsync(string caption, ProgressDialogData initialData, INuGetUI uiService);
+
+        void FireProjectActionsExecuted(IReadOnlyCollection<ProjectAction> projectActions);
```

Comment:
.NET DESIGN GUIDELINES say to stay away from Fire or Trigger, in favor for Raise.
[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/event](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/event)